src/Microsoft.Framework.Runtime/NuGet/NETPortable/NetPortableProfile.cs

```diff
@@ -42,27 +42,27 @@ public NetPortableProfile(string frameworkDirectory, string name, IEnumerable<Fr
         {
             if (String.IsNullOrEmpty(frameworkDirectory))
             {
-                throw new ArgumentNullException( "version");
```

Comment:
This PR does help us capture a bug. We should use `frameworkDirectory` instead of `version` here